src/extensions/Statiq.Html/Globalization.cs

```diff
@@ -0,0 +1,531 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using AngleSharp.Dom.Html;
+using AngleSharp.Parser.Html;
+using Statiq.Common;
+
+namespace Statiq.Html
+{
+    public class Globalization : Module
+    {
+        private enum Direction
+        {
+            LeftToRight,
+            RightToLeft,
+            Neutral
+        }
+
+        protected async override Task<IEnumerable<Common.IDocument>> ExecuteContextAsync(IExecutionContext context)
+        {
+            List<Common.IDocument> list = new List<Common.IDocument>();
+
+            HtmlParser parser = new HtmlParser();
+            foreach (Common.IDocument input in context.Inputs)
+            {
+                IHtmlDocument htmlDocument = await input.ParseHtmlAsync(context, parser);
```

Comment:
Nice use of the `.ParseHtmlAsync()` extension, glad you found that

Reply:
Thanks :)